server/searchHandler.go

```diff
@@ -0,0 +1,74 @@
+package main
```

Comment:
Not sure if this is the preferred way to do this, but idea was to make server.go less-cluttered, and this made testing a bit cleaner too.